samples/boards/arc_secure_services/sample.yaml

```diff
@@ -6,7 +6,7 @@ tests:
   sample.board.arc_secure_services:
     # Requires multiple kernels in an AMP config. See README.rst
     build_only: true
-    platform_allow: nsim_sem em_starterkit_em7d_secure
```

Comment:
Perhaps `em_starterkit_em7d_secure` should be changed just into `em_starterkit_em7d`?

Reply:
@ruuddw  @abrodkin ^^

Reply:
@nashif thanks for pointing to this one. Indeed, back in the day (https://github.com/zephyrproject-rtos/zephyr/pull/9341/) we had that another platform but IIRC we never upstreamed due to missing way of building multi-image application.

Still I'l like to ask @YuguoWH to comment here as well since he's actively working on a similar functionality these days.

Reply:
There's no `em_starterkit_em7d_secure` now, and we should change it into `em_starterkit_em7d`.

Reply:
agreed, em_starterkit_em7d also supports the SecureShield secure mode.